web/src/settings_preferences.ts

```diff
@@ -319,7 +352,7 @@ export function update_page(property) {
     // settings_org.set_input_element_value doesn't support radio
     // button widgets like these.
     if (property === "emojiset" || property === "user_list_style") {
-        $container.find(`input[value=${CSS.escape(value)}]`).prop("checked", true);
+        $container.find(`input[value=${CSS.escape(value.toString())}]`).prop("checked", true);
```

Comment:
`value` (`user_settings[property]`) can be of type `number` too, but `.escape` takes only `string` type.